### DISCUSSION TITLE
B站直播更新之后看不到每日经验了

### DISCUSSION THREAD

**Discussion by Zeddddd** [andywang425/BLTH#500] [Development]
请问有办法重新让每日经验可视化、量化吗？

**Comment by andywang425** [comment]
不太清楚你在说什么，个人中心不是能看到每日经验的吗？

**Comment by Zeddddd** [comment]
是在直播页面，之前鼠标移动到徽章上就能看到每日1500点经验，各自走了多少，现在改版之后不再显示单日的经验进度的多少就不太方便



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年6月29日(周六) 下午2:45
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [andywang425/BLTH] B站直播更新之后看不到每日经验了 (Discussion #500)




 
不太清楚你在说什么，个人中心不是能看到每日经验的吗？

**Comment by andywang425** [comment]
确实，不太懂B站为什么要去掉这个功能。我觉得可以再观望一阵，或许以后就又加上每日经验显示了。
